templates/head-meta.html

```diff
@@ -21,5 +21,5 @@
   window.Polymer = window.Polymer || {dom: 'shadow', lazyRegister: true};
 </script>
 <!-- Node: the webcomponents polyfill is lazily loaded in app.js -->
-<link rel="import" href="/elements/pw-shell.html" async>
+<link rel="import" href="/elements/pw-shell.html">
```

Comment:
`async` was removed because the `custom-style` below uses modules defined in this import (for styling things in the light DOM).